ostd/src/mm/frame/meta.rs

```diff
@@ -543,6 +545,50 @@ fn alloc_meta_frames(tot_nr_frames: usize) -> (usize, Paddr) {
     (nr_meta_pages, start_paddr)
 }
 
+/// The metadata of physical pages that are not usable for the kernel.
+#[derive(Debug)]
+pub struct UnusableMemoryMeta;
+impl_frame_meta_for!(UnusableMemoryMeta);
+
+/// The metadata of physical pages that I/O devices use.
+#[derive(Debug)]
+pub struct IOMemoryMeta;
```

Comment:
To follow Rust's naming convention for abbreviations, rename this type to `IoMemoryArea`.